tests/package.json

```diff
@@ -38,67 +38,17 @@
     "@react-native/babel-preset": "^0.74.84",
     "@react-native/metro-config": "^0.74.84",
     "axios": "^1.5.1",
-    "detox": "^19.13.0",
+    "detox": "20.23.1",
```

Comment:
magic!

Reply:
This was a fire fight for sure, especially switching Mocha to Jet